mandatory/4-tax.js

```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(productPrice) {
+let productTax = (productPrice*20) / 100 + productPrice;
```

Comment:
This is more of an advanced comment, but it's usually bad to have 'magic numbers' in code i.e. any numbers that don't have an obvious meaning. So I'd advise using an extra line to explain that the `20` is the tax rate (the 100 is okay because it's obvious in the code that you're using it to make a percentage):

```
function calculateSalesTax(originalPrice) {
  const taxRate = 20;
  const taxOnOriginal = originalPrice * (taxRate / 100);
  return originalPrice + taxOnOriginal;
}
```